### ISSUE TITLE
Is this still working like image donwloading ??

### ISSUE THREAD

**Issue by sunil-singh-chaudhary** [bachors/Insta-Downloader#21]
hi ,
How can i get accessToken your Link seem not working ok if I get accessToken is it possible to donwload images and videos pls reply anybody get any success here

**Comment by banyanstudio** [comment]
Same question!..Did you resolve it?

**Comment by sunil-singh-chaudhary** [comment]
yes but i didnt use this method because these are blocked by instagram because of change in API's

**Comment by sunil-singh-chaudhary** [comment]
i made a demo for you check 
it  happy CODING

https://github.com/sunil-singh-chaudhary/InstagramPic-VideoDownloader
